internal/tenant/model.go

```diff
@@ -2,35 +2,44 @@ package tenant
 
 import "time"
 
-type TenantInfoList struct {
-	Data []TenantInfoData
+// InfoList defines list of informations about a tenant
+type InfoList struct {
+	Data []InfoData
 	Meta struct {
 		TotalCount int
 	}
 	Errors []Error `json:"errors"`
 }
-type TenantInfo struct {
-	Data   TenantInfoData
+
+// Info defines a single tanent information
+type Info struct {
+	Data   InfoData
 	Errors []Error `json:"errors"`
 }
 
+// Error defines an http error
 type Error struct {
 	Code   string `json:"code"`
 	Detail string `json:"detail"`
 }
 
-type TenantInfoData struct {
+// InfoData is Data used in Info and InfoList
+type InfoData struct {
 	Attributes Attributes
-	Id         string
+	ID         string
 	Type       string
 }
 
+// Attributes is Attributes used in InfoData
```

Comment:
are attributes

Reply:
Attributes and InfoData are struct names